server/server_test.go

```diff
@@ -0,0 +1,49 @@
+package server_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/gin-gonic/gin"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/parkmenow/PMN-api/server"
```

Comment:
An extra line space is required between someone else's files and our files

Reply:
Created new commit to resolve this d4a3cc87ced24e90be62f2359efe80fb408a71c9